Packages/manifest.json

```diff
@@ -1,21 +1,13 @@
 {
   "dependencies": {
-    "com.madsbangh.easybuttons": "https://github.com/madsbangh/EasyButtons.git#upm",
-    "com.unity.2d.animation": "4.2.6",
-    "com.unity.2d.pixel-perfect": "3.0.2",
-    "com.unity.2d.psdimporter": "3.1.6",
-    "com.unity.2d.sprite": "1.0.0",
-    "com.unity.2d.spriteshape": "4.1.4",
-    "com.unity.2d.tilemap": "1.0.0",
-    "com.unity.collab-proxy": "1.3.9",
-    "com.unity.ide.rider": "1.2.1",
-    "com.unity.ide.visualstudio": "2.0.3",
+    "com.unity.cinemachine": "2.6.3",
+    "com.unity.collab-proxy": "1.2.16",
+    "com.unity.ide.rider": "1.1.4",
     "com.unity.ide.vscode": "1.2.3",
     "com.unity.inputsystem": "1.0.2",
-    "com.unity.render-pipelines.universal": "8.2.0",
-    "com.unity.test-framework": "1.1.18",
-    "com.unity.textmeshpro": "3.0.1",
-    "com.unity.timeline": "1.3.6",
+    "com.unity.test-framework": "1.1.22",
```

Comment:
Hey if possible don't downgrade packages

Reply:
I don't even know how I did that.